desktop/cmp/grid/impl/ColChooserModel.js

```diff
@@ -60,7 +60,7 @@ export class ColChooserModel {
             }
         });
 
-        gridModel.applyColumnChanges(colChanges);
+        gridModel.applyColumnStateChanges(colChanges);
```

Comment:
This doesn't relate directly to this change, but would there be any value in adjusting this method to only add columns to `colChanges` if the visibility changed? It seems like that would be more appropriate, and potentially avoid triggering the additional logic within gridModel related to sort order.